cli/cmd/honeyvent.go

```diff
@@ -218,3 +219,18 @@ func installMethod() string {
 	}
 	return ""
 }
+
+// parseFlags is a helper used to parse all the flags that the user provided
+func parseFlags(args []string) (flags []string) {
+	for len(args) > 0 {
+		arg := args[0]
+		args = args[1:]
+		if len(arg) == 0 || arg[0] != '-' || len(arg) == 1 {
```

Comment:
nit can reduce this. 
```suggestion
		if len(arg) <= 1 || arg[0] != '-' {
```